pkg/pb/contacts.proto

```diff
@@ -21,16 +22,19 @@ message Contact {
     string last_name = 4;
     string primary_email = 6 [(gorm.field).drop = true, (validate.rules).string.email = true];
     repeated Email emails = 7;
+    // nicknames is arbitrary json, but should be used for a list of strings
+    gorm.types.JSONValue nicknames = 8;
 }
 
 message Email {
     option (gorm.opts) = {
       ormable: true,
+      multi_account: true,
       include: [
       {type: "bool", name: "is_primary"}, {type: "uint64", name: "Contact_id"}]
     };
     uint64 id = 1;
-    string address = 2 [(gorm.field).tags = "gorm:\"UNIQUE\"", (validate.rules).string.email = true];
+    string address = 2 [(gorm.field).tag.unique = true, (validate.rules).string.email = true];
```

Comment:
does adding the `unique` tag here make gorm handle it any differently (besides adding the tag to the model definition)?

Reply:
No, there's no special code for this beyond adding the `UNIQUE` tag to the gorm model.